gravitl/netclient
# added windows gateway detection

To test:
Join a network with a Windows node, ensure the host's DefaultGateway and LocalAddress are set appropriately.
